content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+          "data": {
+            "id": "provver-y5KZUsSBRLV9zCtL",
+            "type": "registry-provider-versions"
+          }
+        }
+      },
+      "links": {
+        "provider-binary-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+      }
+    },
+    {
+      "id": "provpltfrm-qQYosUguetYtXGzJ",
+      "type": "registry-provider-platforms",
+      "attributes": {
+        "os": "linux",
+        "arch": "arm64",
+        "filename": "terraform-provider-aws_3.1.1_linux_arm64.zip",
+        "shasum": "d6b668151b6d2f9826093ca1d7b6f54a1cf00353340e20f5f238c99a8fbb727e",
+        "permissions": {
+          "can-delete": true,
+          "can-upload-asset": true
+        },
+        "provider-binary-uploaded": true
+      },
+      "relationships": {
+        "registry-provider-version": {
+          "data": {
+            "id": "provver-y5KZUsSBRLV9zCtL",
+            "type": "registry-provider-versions"
+          }
+        }
+      },
+      "links": {
+        "provider-binary-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+      }
+    },
+    {
+      "id": "provpltfrm-pjDHFN46y193bS7t",
+      "type": "registry-provider-platforms",
+      "attributes": {
+        "os": "linux",
+        "arch": "fakearch",
+        "filename": "terraform-provider-aws_3.1.1_linux_amd64.zip",
+        "shasum": "8f69533bc8afc227b40d15116358f91505bb638ce5919712fbb38a2dec1bba38",
+        "permissions": {
+          "can-delete": true,
+          "can-upload-asset": true
+        },
+        "provider-binary-uploaded": true
+      },
+      "relationships": {
+        "registry-provider-version": {
+          "data": {
+            "id": "provver-y5KZUsSBRLV9zCtL",
+            "type": "registry-provider-versions"
+          }
+        }
+      },
+      "links": {
+        "provider-binary-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+      }
+    }
+  ],
+  "links": {
+    "self": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "first": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "prev": null,
+    "next": null,
+    "last": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms?page%5Bnumber%5D=1&page%5Bsize%5D=20"
+  },
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "page-size": 20,
+      "prev-page": null,
+      "next-page": null,
+      "total-pages": 1,
+      "total-count": 5
+    }
+  }
+}
+```
+
+### Note
+
+`provider-binary-uploaded` may be `false` if that file hasn't been uploaded to Archivist yet. In this case, instead of including a link to `provider-binary-download`. The response will include an upload link `provider-binary-upload` instead.
```

Comment:
```suggestion
### Note

-> **Note:** The `provider-binary-uploaded` will be `false` if that file hasn't been uploaded to Archivist yet. In this case, instead of including a link to `provider-binary-download`, the response will include an upload link `provider-binary-upload`.
```